testsuite/tests/operator/test_apicast_operator_labels.py

```diff
@@ -0,0 +1,57 @@
+"""
+Check if labels of apicast operator pod are present
+"""
+from typing import Tuple, List, Union
+
+import pytest
+from packaging.version import Version
+
+from testsuite import TESTED_VERSION
+from testsuite.capabilities import Capability
+
+pytestmark = [
+    pytest.mark.issue("https://issues.redhat.com/browse/THREESCALE-7751"),
+    pytest.mark.required_capabilities(Capability.OCP4),
+]
+
+LABELS_pre_2_12: List[Union[Tuple[str, str], Tuple[str, None]]] = [
+    ("app", "apicast"),
+    ("control-plane", "controller-manager"),
+]
+
+LABELS_post_2_12: List[Union[Tuple[str, str], Tuple[str, None]]] = [
+    ("app", "apicast"),
+    ("com.company", "Red_Hat"),
+    ("control-plane", "controller-manager"),
+    ("rht.comp", "3scale_apicast"),
+    ("rht.comp_ver", None),
+    ("rht.prod_name", "Red_Hat_Integration"),
+    ("rht.prod_ver", None),
+    ("rht.subcomp", "apicast_operator"),
+    ("rht.subcomp_t", "infrastructure"),
+]
+
+if TESTED_VERSION >= Version('2.12'):
```

Comment:
I would like more the solution using `parameterize` with `pytest.param` and skipif.